packages/providers/src/lib/push/one-signal/one-signal.provider.ts

```diff
@@ -38,8 +46,17 @@ export class OneSignalPushProvider
   ): Promise<ISendMessageSuccessResponse> {
     const { sound, badge, ...overrides } = options.overrides ?? {};
 
+    const targetSegment =
+      this.apiVersion === 'externalId'
+        ? {
+            include_aliases: {
+              external_id: options.target,
+            },
+          }
+        : { include_player_ids: options.target };
+
     const notification = this.transform(bridgeProviderData, {
```

Comment:
@L-U-C-K-Y, do you know by chance if the API body signature is similar to both modes and API endpoints?

Reply:
I'm just wondering if we even need to change the BASE URL at all

Reply:
@scopsy yes that would still be required if we want to keep "include_player_ids" in the older model.

See here the API Documentation for v9 (old):
https://documentation.onesignal.com/v9.0/reference/create-notification

<img width="347" alt="image" src="https://github.com/user-attachments/assets/00a4df0f-9966-4668-b1d8-c83e32869c2d">

V9 does **not** have `include_aliases` (was named: `include_external_user_ids` and still had the concept of player ids and therefore had `include_player_ids`.

And here 11.6 (new):
https://documentation.onesignal.com/reference/push-notification

<img width="360" alt="image" src="https://github.com/user-attachments/assets/c1cb7fb9-c9f6-49a2-8f34-d194791c6059">

V11.6 has `include_aliases` where we can send the external ids, but **no** `include_player_ids`.
In the new versions, player ids are not used anymore and the recommended way is the external id:

https://documentation.onesignal.com/docs/users

_It is always recommended to use the External ID to identify users because this is the only way to track a user across all their subscriptions._